platform-sdk/docs/proposals/template-2.md

```diff
@@ -0,0 +1,184 @@
+# Proposal Title Goes Here
```

Comment:
Is the idea to have two different templates? I'm confused by a `template-2.md` rather than changing the existing one.

Reply:
I did that because it was not clear to me what to do with the result of an "old" proposal. Well, the template is not really a proposal, so here it makes sense to just change it. Will do that.

Reply:
I agree with editing the template in the original file.  It makes it clear as to what has changed through github's automated diff.  Right now I would need to read both and pay close attention to determine if anything has changed.